modules/rest/src/main/java/org/apache/ignite/internal/rest/netty/RestApiInitializer.java

```diff
@@ -46,7 +45,6 @@ public RestApiInitializer(Router router) {
     public void initChannel(SocketChannel ch) {
         ChannelPipeline p = ch.pipeline();
         p.addLast(new HttpServerCodec());
-        p.addLast(new HttpServerExpectContinueHandler());
```

Comment:
Hmm.. don't we need Expect/Continue anymore? Is this somehow connected with the introduced asynchrony?

Reply:
Discussed in person: "continue" is handled by the `HttpObjectAggregator` and the javadoc explicitly states not to use this handler